Add release notes for upcoming 32-bit floating points (#92)
tableau/hyper-db: website/docs/sql/datatype/index.md

`CHARACTER [ (n) ]`|`CHAR [ (n) ]`|fixed-length character string
`CHARACTER VARYING (n)`|`VARCHAR (n)`|variable-length character string with limit
`DATE`||calendar date (year, month, day)
`REAL`|`FLOAT4`|single precision floating-point number (4 bytes)

> `DOUBLE PRECISION`|`FLOAT8`|double precision floating-point number (8 bytes)

Reviewer: Do we want to mention `FLOAT` here?

Author: 86d12d7ad9f81cd1d90835797b9a8a79fd93f48d

`TIMESTAMP WITH TIME ZONE`|`TIMESTAMPTZ`|date and time, including time zone
`GEOGRAPHY`||a geography object

:::note

> Persisting `NUMERICs` with a precision greater than 18 requires at least [database version 3](/docs/hyper-api/hyper_process#version-3).

Reviewer: ```suggestion
Persisting `NUMERIC`s with a precision greater than 18 requires at least [database version 3](/docs/hyper-api/hyper_process#version-3).
```

Author: 86d12d7ad9f81cd1d90835797b9a8a79fd93f48d

:::
:::note

> Persisting 32-bit `REALs` requires at least [database version 4](/docs/hyper-api/hyper_process#version-4).

Reviewer: ```suggestion
Persisting 32-bit floating point values (e.g., type `REAL`) requires at least [database version 4](/docs/hyper-api/hyper_process#version-4).
```

Author: 86d12d7ad9f81cd1d90835797b9a8a79fd93f48d

> Previously, all floating points were mapped to 64-bit `double precision`.

Reviewer 2: Whether a `real` is 32-bit or 64-bit does not depend on the database version, but on the Hyper version...

> Up until Hyper API release [0.0.18825](/docs/releases#0.0.18825) Hyper mapped all floating points internally to 64-bit `DOUBLE PRECISIONs`.

Reviewer: ```suggestion
Up until Hyper API release [0.0.18825](/docs/releases#0.0.18825) Hyper used 64-bit floating points for all float types (i.e., also for `REAL`).
```

Author: 86d12d7ad9f81cd1d90835797b9a8a79fd93f48d
